### DISCUSSION TITLE
Livewire Blade Select2 error message not working without wire:ignore

### DISCUSSION THREAD

**Discussion by omersavas** [livewire/livewire#2810] [1. Help]
laravel version 7.4 > 
livewire 2.3 >

blade page 
 <div class="form-row" >
                <div class="col" wire:ignore>
                    <div class="form-group"  >
                        <label for="">Keyword Types</label>
                        <select wire:model='article.keyword_type' class="form-control select2 @error('article.keyword_type')is-invalid @enderror">
                            <option selected>Choose Keyword</option>
                            @foreach ($keywordTypes as $item)
                                <option value="{{ $item['id'] }}" >{{ $item['name'] }}</option>
                            @endforeach
                        </select>     
                                   
                    </div>
                </div>
                @error('article.keyword_type')
                <div class="invalid-feedback">
                    {{ $message }}
                </div>
                @enderror    
            </div>

Backend rules
 protected $rules = [
        'article.name'                  => 'required',
        'article.content'               => 'required',
        'article.directory_id'          => 'required',
        'article.keyword_type'          => 'required',
    ];

save function call   $this->validate(); function normal inputs is work error message but select without wire:ignore not show

**Comment by adeiming** [comment]
@omersavas have you read this documentation: [https://laravel-livewire.com/docs/2.x/alpine-js](https://laravel-livewire.com/docs/2.x/alpine-js)

I am using aplin.js to handle other js libraries, and it is working fine

**Comment by hitoridev** [comment]
@adeiming i have the same issue, bro how do handle livewire validation error not showing when select2 div attached with wire:ignore?

**Comment by joshhanley** [comment]
@omersavas and @hitoridev you should put the error message outside the div that has `wire:ignore` on it.

It's hard to see exactly what is going on @omersavas  as there is no select or wire:ignore in the code that has been provided.

Please provide all the necessary code and format it in code blocks correctly so we can have a look.

As I don't think this is a Livewire bug, I'm going to convert it to a discussion for now.

Hope this helps!
